### PR TITLE
fixing links to close issue #544

### PR DESCRIPTION
**Pull request by sdwheeler**
Update to WMF scenario links

### CODE REVIEW

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[wmf/5.1/scenarios-features.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue544/wmf/5.1/scenarios-features.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/scenarios-features?branch=pr-en-us-2620) | [Details](#user-content-wmf%2F5.1%2Fscenarios-features.md)

<a name="wmf%2F5.1%2Fscenarios-features.md"></a>
### [wmf/5.1/scenarios-features.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue544/wmf/5.1/scenarios-features.md)
 - **Line 55**: **[Warning]** Invalid file link:(~/5.1/powershell/module/Microsoft.PowerShell.Security/Set-AuthenticodeSignature).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C26%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806261701008501-2620%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=34O2ssYK1ZOJ%2BFBzXld1AQbHqhu3DuX53vnTxcA0zRs%3D&st=2018-06-26T17%3A07%3A22Z&se=2018-07-27T17%3A12%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: wmf/5.1/scenarios-features.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ This creates the catalog file.
 
 ![](../images/CatalogFile2.jpg)
 
-To verify the integrity of catalog file (Pester.cat in above example), sign it using [Set-AuthenticodeSignature](https://technet.microsoft.com/library/hh849819.aspx) cmdlet.
+To verify the integrity of catalog file (Pester.cat in above example), sign it using [Set-AuthenticodeSignature](powershell/module/Microsoft.PowerShell.Security/Set-AuthenticodeSignature) cmdlet.
```
Comment (DCtheGeek) [inline wmf/5.1/scenarios-features.md]: Missing forward slash in front of link.

File: wmf/5.1/scenarios-features.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ Previously, you had no way to specify a particular module version; if there were
 
 In WMF 5.1:
 
-- You can use [ModuleSpecification Constructor (Hashtable)](https://msdn.microsoft.com/library/jj136290).
+- You can use [ModuleSpecification Constructor (Hashtable)](https://docs.microsoft.com/en-us/dotnet/api/microsoft.powershell.commands.modulespecification.-ctorview=powershellsdk-1.1.0#Microsoft_PowerShell_Commands_ModuleSpecification__ctor_System_Collections_Hashtable_).
```
Comment (DCtheGeek) [inline wmf/5.1/scenarios-features.md]: I believe we don't need the FQDN, just the /dotnet/api/ and on.

Comment (sdwheeler) [reply wmf/5.1/scenarios-features.md]: missed that - fixed

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[wmf/5.1/scenarios-features.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue544/wmf/5.1/scenarios-features.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/scenarios-features?branch=pr-en-us-2620) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C26%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806261713451076-2620%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=2zPiV%2BWpth5E7fK%2BggSGUOzJlFcqCzSlBF%2FPV00sTmw%3D&st=2018-06-26T17%3A19%3A31Z&se=2018-07-27T17%3A24%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[wmf/5.1/scenarios-features.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue544/wmf/5.1/scenarios-features.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/scenarios-features?branch=pr-en-us-2620) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C26%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806261718178072-2620%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=p%2B%2BR%2FXFqD6F11tWft0YbX0yLhgXZBOHhJcKEK%2FUQd4k%3D&st=2018-06-26T17%3A30%3A33Z&se=2018-07-27T17%3A35%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[wmf/5.1/scenarios-features.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue544/wmf/5.1/scenarios-features.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/wmf/5.1/scenarios-features?branch=pr-en-us-2620) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C26%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806261739439173-2620%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=yOd8RRuwO9OIKhP4ug7rw7R%2BvvnbGGDKyDauRsnQj2g%3D&st=2018-06-26T17%3A45%3A10Z&se=2018-07-27T17%3A50%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sdwheeler** [comment]
#sign-off Preview LGTM

**Pull request merged**